src/crimson/tools/store_bench/store-bench.cc

```diff
@@ -676,10 +673,12 @@ seastar::future<results_t> RandomWriteWorkload::run(
    coll_refs.emplace_back(std::make_pair(cid, std::move(ref)));
   }
   auto get_coll_id = [&](uint64_t obj_id) {
-    return coll_refs[obj_id % get_obj_per_coll()].first;
+    assert(obj_id % colls_per_shard < coll_refs.size());
+    return coll_refs.at(obj_id % colls_per_shard).first;
```

Comment:
The assert above won't verify that this index actually exists, right? `at` throws an error in this case.
What is the reason not to assert the specific index instead?

Reply:
We could switch to unordered_map